core-default/pom.xml

```
@@ -5,24 +5,22 @@
     <parent>
         <artifactId>beetle</artifactId>
         <groupId>me.aurium</groupId>
-        <version>0.2.0</version>
+        <version>1.1.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <version>0.3.4</version>
-
     <artifactId>beetle-core-default</artifactId>
     <dependencies>
         <dependency>
             <groupId>me.aurium</groupId>
             <artifactId>beetle-core</artifactId>
-            <version>${core.version}</version>
         </dependency>
+
         <!-- this is provided because... only use hikari shit if you want it !-->
         <dependency>
             <groupId>com.zaxxer</groupId>
             <artifactId>HikariCP</artifactId>
-            <version>4.0.1</version>
+            <version>4.0.3</version>
             <scope>provided</scope>
```

```suggestion
            <scope>optional</scope>
```
You can do the same for slf4j-api and jooq, if they are really optional.

---

I didn't even know that was a scope i could use!

---

I'm not sure if slf4j is optional - I intend for it to be but unintentionally could be invoking the loggerfactory somewhere - something to keep an eye out for when reviewing !

---

As it would turn out, slf4j is not optional! (LMFAO)

---

Uhh I didn't mean to write `<scope>optional</scope>`. It should be `<optional>true</optional>`.

You are right that slf4j is not an optionally dependency of Beetle.